compiler/rustc_interface/src/queries.rs

```diff
@@ -83,7 +82,6 @@ pub struct Queries<'tcx> {
     register_plugins: Query<(ast::Crate, Lrc<LintStore>)>,
     expansion: Query<(ast::Crate, Steal<Rc<RefCell<BoxedResolver>>>, Lrc<LintStore>)>,
     dep_graph: Query<DepGraph>,
-    lower_to_hir: Query<(&'tcx Crate<'tcx>, Steal<ResolverOutputs>)>,
```

Comment:
Perhaps Zoxc had something in mind when making `lower_to_hir` individually available, but if no tools use it right now, then it seems fine to remove.